### ISSUE TITLE
package test data in an ilastik-tests package

### ISSUE THREAD

**Issue by k-dominik** [ilastik/ilastik#2771]
Currently we use the conda build system to build the packages `ilastik`, `ilastik-gpu`, `ilastik-core`. All of these include the test data. Uncompressed our test ilastik folder looks like this:

```
 92M	tests/test_ilastik
 87M	tests/test_ilastik/data
4.1M	tests/test_ilastik/test_applets
792K	tests/test_ilastik/test_workflows
312K	tests/test_ilastik/widgets
...
```
clearly, the biggest part is is the data. For releases it gets included twice (`ilastik`, `ilastik-gpu` both depend on `ilastik-core`).

It is not possible to exclude from the conda package with simple means, see discussion [here](https://github.com/conda/conda-build/issues/3451).

One of the suggestions is to build a dedicated package for the test data.

Pros:
* conda package will be much lighter - faster to install
* releases will get almost 200mb slimmer

Cons:
* might make development more awkward.
* probably a lot of work to adapt our tests to work from that. (We use relative paths from `__file__` a lot)

**Comment by k-dominik** [comment]
I'm gonna 👍 myself, as I had this idea again and already wanted to open a new issue ;D

**Comment by k-dominik** [comment]
maybe [pooch](https://pypi.org/project/pooch/) could be an option to manage these data (thx @FynnBe)

**Comment by FynnBe** [comment]
At danger of pointing out the obvious.... with pooch you wouldn't need a dedicated package. You could load the same collection from multiple projects/packages.

**Comment by k-dominik** [comment]
yeah thank you Fynn :) I guess the title here does not reflect the sentiment. In the end I want a slimmer ilastik package - dedicated test package was the idea of November...
